<launch>
  <arg name="launch_prefix" default="" />

  <node pkg="hobot_vio" type="hobot_vio" name="hobot_vio" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="image_topic" type="string" value="/camera/color/image_raw" />
    <param name="imu_topic" type="string" value="/camera/imu" />
    <param name="sample_gap" type="int" value="2" />
  </node>

  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="enable_depth" value="false"/>
    <arg name="enable_color" value="true"/>
    <arg name="enable_gyro" value="true"/>
    <arg name="enable_accel" value="true"/>
    <arg name="enable_sync" value="true"/>
    <arg name="enable_confidence" value="false"/>
    <arg name="publish_odom_tf" value="false"/>
    <arg name="gyro_fps" value="200"/>
    <arg name="accel_fps" value="200"/>
    <arg name="color_fps" value="30"/>
    <arg name="unite_imu_method" value="linear_interpolation"/>
    <arg name="color_width" value="640"/>
    <arg name="color_height" value="480"/>
  </include>
</launch>
